packages/runtime/test/consumption/attributes.test.ts

```diff
@@ -1892,7 +1895,7 @@ describe("DeleteAttributeUseCases", () => {
             content: {
                 value: {
                     "@type": "GivenName",
-                    value: "Petra Pan"
+                    value: "Petro Pan"
```

Comment:
```suggestion
                    value: "aGivenName"
```

Reply:
please use givenName and anotherGivenName - we don't use ANY of those fakish names.